src/gourmand/importers/web_importer.py

```diff
@@ -41,7 +41,7 @@ def import_urls(urls: List[str]) -> Tuple[List[str], List[str]]:
             urls.remove(url)
 
     for url in urls:
-        recipe = scrape_me(url)
+        recipe = scrape_html(html,org_url=url)
```

Comment:
This does not look correct - `html` is undefined here.

```suggestion
        recipe = scrape_html(html=None, org_url=url)
```

Reply:
Providing html is optional see https://github.com/hhursev/recipe-scrapers/blob/main/recipe_scrapers/__init__.py#L892

Reply:
Perhaps it is better to download the html separately as online parameter will be removed:
https://github.com/hhursev/recipe-scrapers/blob/main/recipe_scrapers/__init__.py#L937

Reply:
While `html` can be a string or `None`, we still have to provide it. For the other refactoring, feel free to change it here or submit a PR later on.

Reply:
Can submit a PR later. Created an issue https://github.com/GourmandRecipeManager/gourmand/issues/176